steps/checkmarx/0.1.0/step.yml

```diff
@@ -0,0 +1,132 @@
+title: Checkmarx SAST Analysis
+summary: Performs static code analysis using the Checkmarx SAST (https://checkmarx.com).
+description: |
+  Performs static code analysis using the Checkmarx SAST (https://checkmarx.com).
+website: https://checkmarx.com
+source_code_url: https://github.com/miguelfreitas93/bitrise_checkmarx_step
+support_url: https://github.com/miguelfreitas93/bitrise_checkmarx_step/issues
+source:
+  git: https://github.com/miguelfreitas93/bitrise_checkmarx_step.git
+  commit: 75575075e2554b4c26550ddc476640bd1b838415
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+project_type_tags:
+- ios
+- xamarin
+- cordova
+- ionic
+- react-native
+- android
+type_tags:
+  - test
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: true
+run_if: ""
+toolkit:
+  go:
+    package_name: github.com/miguelfreitas93/bitrise_checkmarx_step
+inputs:
+  - bitrise_source_dir: "$BITRISE_SOURCE_DIR"
+    opts:
+      title: Path to the source code folder
+      description: |
+        Recommended is $BITRISE_SOURCE_DIR.
+      is_expand: true
+      is_required: true      
+  - cx_server_url: 
+    opts:
+      title: Checkmarx Server URL
+      description: |
+        URL to access Checkmarx Server (https://www.checkmarx.company.com or http://10.10.0.21:443)
+      is_expand: true
+      is_required: true
+  - cx_username: 
+    opts:
+      title: CX Username
+      is_expand: true
+      is_required: true
+      is_sensitive: true
+  - cx_password: 
+    opts:
+      title: CX Password
+      is_expand: true
+      is_required: true
+      is_sensitive: true
+  - cx_project_name: 
+    opts:
+      title: Project Name
+      description: |
+        Project Name, for example: BitriseTest
+      is_expand: true
+      is_required: true
+      is_sensitive: true
+  - cx_team_name: 
+    opts:
+      title: Team Name
+      description: |
+        Team Name, for example: \\CxServer\\SP\\Company
+      is_expand: true
+      is_required: true
+      is_sensitive: true
+  - cx_preset_name: 
+    opts:
+      title: Preset Name
+      description: |
+        Preset Name, for example: Mobile
+      is_expand: true
+      is_required: true
+      is_sensitive: true
+  - cx_exclude_folders: 
+    opts:
+      title: Exclude Folders
+      description: |
+        Exclude Folders, for example: *test, *mock*, example*
+      is_expand: true
+      is_required: false
+      is_sensitive: true
```

Comment:
This shouldn't be marked as sensitive perhaps.
Sensitive is mainly for credentials (passwords, access tokens etc.)

Reply:
Hi Koral, 
You're right, I thought "is_sensitive" meant case sensitive.
Thanks !